source/cgame/cg_local.h

```diff
@@ -373,6 +374,8 @@ typedef struct
 	byte_vec4_t color;
 	struct shader_s *icon;
 	int modelindex;
+	uint64_t steamid;
+	unsigned char avatar[32 * 32 * 4]; // 32x32, 4 channels
```

Comment:
Why embed the picture like this?
How can you see if it's initialized this way?

Just register a picture in the gfx subsystem when you receive an avatar, and drop the picture handle or number or whatever you get back in here.

Reply:
ah thanks, i was looking through the graphics surface for a while that seems to work

Reply:
Don't forget to unregister it when a user leaves